public/modules/articles/articles.viz.js

```diff
@@ -499,6 +515,9 @@ var valign, stacked, articlesScope, aligned = true;
                                         .popover(function () {
                                             return section_hover(d, curS)
                                         })
+                                        .on("mouseenter", function() {
+                                            $(".popover").addClass("articles-popover");
+                                        })
                                         .text(clean_premier(titre_section(sub_section(curS), longlabel)));
                                     ct++;
```

Comment:
*Related: we should definitely get rid of all bootstrap popovers/tooltips. They're a PITA in terms of performance. And bootstrap does nearly everything wrong with them.*